packages/portal/src/components/item/ItemMediaPresentation.vue

```diff
@@ -422,6 +424,7 @@
 
   .media-viewer-wrapper {
     position: relative;
+    background-color: $black;
```

Comment:
Revert this to be scoped to `.media-viewer-inner-wrapper`. Otherwise the black bg will show briefly when closing the thumbnails menu on mobile where that looks a bit off.